__mocks__/handlers/task-details.handler.ts

```diff
@@ -34,18 +35,29 @@ const taskDetailsHandler = [
         );
     }),
 ];
+
+const taskDetailsUpdateHandler = [
```

Comment:
I think we don't need a separate variable for the patch request and this can be added in the array above.

Reply:
Moved it into the TaskdetailsHandler array 👍🏻